README.md

```diff
@@ -17,17 +17,10 @@ A chat sharing application built with Python.
 
 ## Running the Application
 
-Simply run the Python script:
+Run the application as a module:
 
 ```bash
-python3 chatshare.py
-```
-
-Or if you want to make it executable:
-
-```bash
-chmod +x chatshare.py
-./chatshare.py
+python3 -m src.chatshare
```

Comment:
Is it supposed to be `src.chatshare` or `src/chatshare`

Reply:
`src.chatshare`. the -m means it's running as a module rather than a file. I tried `python3 src/chatshare` and it didn't work :(